sklearn/cross_decomposition/pls_.py

```diff
@@ -367,6 +370,11 @@ def fit(self, X, Y):
             # => B = W*Q' (p x q)
             self.coef_ = np.dot(self.x_rotations_, self.y_loadings_.T)
             self.coef_ = self.coef_ * self.y_std_
+
+            if Y_ndim_is_one:
+                self.coef_ = np.ravel(self.coef_)
```

Comment:
Does `LinearRegression` do something similar? I am wondering whether this is the right fix or whether we should just change the logic in `predict` ...